manifests/ca.pp

```diff
@@ -23,10 +23,6 @@
 ) {
   $server_ca_path = "${certs::ssl_build_dir}/${server_ca_name}.crt"
 
-  file { "${certs::pki_dir}/private/${default_ca_name}.pwd":
-    ensure => absent,
```

Comment:
This was ensured absent in 4ba477a68860d1ce7085699020a83755f066debf, released in 17.0.0.